proto/comdex/lend/v1beta1/gov.proto

```diff
@@ -31,6 +31,12 @@ message AddAssetToPairProposal {
   AssetToPairMapping AssetToPairMapping = 3 [(gogoproto.nullable) = false];
 }
 
+message AddMultipleAssetToPairProposal {
+  string title = 1 [(gogoproto.moretags) = "yaml:\"title\""];
+  string description = 2 [(gogoproto.moretags) = "yaml:\"description\""];
+  repeated AssetToPairSingleMapping AssetToPairSingleMapping = 3 [(gogoproto.nullable) = false];
```

Comment:
snake_case for variable names

Reply:
We will be renaming it in future refactoring.